commands/server.go

```diff
@@ -446,8 +448,15 @@ func (c *commandeer) serve(s *serverCmd) error {
 		mu, serverURL, endpoint, err := srv.createEndpoint(i)
 
 		if doLiveReload {
-			mu.HandleFunc("/livereload.js", livereload.ServeJS)
-			mu.HandleFunc("/livereload", livereload.Handler)
+			u, err := url.Parse(baseURLs[i])
+			if err != nil {
+				return err
+			}
+			if !strings.HasSuffix(u.Path, "/") {
+				u.Path += "/"
+			}
+			mu.HandleFunc(u.Path+"livereload.js", livereload.ServeJS)
+			mu.HandleFunc(u.Path+"livereload", livereload.Handler)
```

Comment:
considering the repeated checking for trailing slashes here and below, then concatenating a string.
Would using the utility `helpers.SanitizeURL(string) string` be more appropriate? at that point, as it strips trailing slashes, all the string literals may start with a slash for the same effect.

Though it may instead be better to ensure the baseURL flag gets this treatment at the command entrypoint rather than down here

Reply:
`SanitizeURL()` is definitely a good idea